tasks/image-orientation/main.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from coretex import currentTaskRun, Model, folder_manager, Metric, MetricType, ImageDataset, TaskRun
+from torch.utils.data import DataLoader
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+from src.dataset import OrientedDataset, getTransform, prepareDataset, splitDataset
+from src.model import OrientationClassifier
+from src.train import runTraining
+from src.validation import runValidation
+from src.utils import getMeanAndStd
+
+
+def createMetrics(taskRun: TaskRun) -> None:
+    taskRun.createMetrics([
+        Metric.create("training_loss", "epoch", MetricType.int, "value", MetricType.float, [1, 10]),
+        Metric.create("validation_loss", "epoch", MetricType.int, "value", MetricType.float, [1, 10]),
+        Metric.create("training_accuracy", "epoch", MetricType.int, "value", MetricType.float, [1, 10]),
+        Metric.create("validation_accuracy", "epoch", MetricType.int, "value", MetricType.float, [1, 10])
+    ])
+
+
+def main():
+    taskRun: TaskRun[ImageDataset] = currentTaskRun()
+    createMetrics(taskRun)
+    taskRun.dataset.download()
+
+    logging.info(">> [Orientation] Preparing dataset")
+    imagesDir, sampleIds = prepareDataset(taskRun.dataset)
+
+    imageSize = taskRun.parameters["imageSize"]
+    epochs = taskRun.parameters["epochs"]
+    batchSize = taskRun.parameters["batchSize"]
+    labelColumn = taskRun.parameters["labelColumn"]
+
+    mean, std = getMeanAndStd(imagesDir)
+
+    dataset = OrientedDataset(imagesDir, sampleIds, labelColumn, transform = getTransform((imageSize, imageSize), (mean, std)))
```

Comment:
export getTransform(..) into constant

Reply:
How about I create a variable called "transform" after the getMeanAndStd function call instead of a constant?